packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -1865,6 +1863,41 @@ function useBorderBoxResizeObserver(
   }, [onResize, ref]);
 }
 
+function usePrefersReducedMotion(nodeRef: React.RefObject<Element>) {
+  const [prefersReducedMotion, setPrefersReducedMotion] = React.useState(false);
+
+  React.useEffect(() => {
+    const globalWindow = nodeRef.current?.ownerDocument.defaultView || window;
+
+    try {
+      function handleChange(event: MediaQueryListEvent) {
+        setPrefersReducedMotion(!event.matches);
+      }
+      const mediaQueryList = globalWindow.matchMedia('(prefers-reduced-motion: no-preference)');
+      mediaQueryList.addEventListener('change', handleChange);
+      return function () {
+        mediaQueryList.removeEventListener('change', handleChange);
+      };
+    } catch (e) {}
+
+    return function () {};
```

Comment:
I know this wasn't you but if we're simplifying, why do we need the `try`/`catch` and the empty `return function () {}` here?

```suggestion
    function handleChange(event: MediaQueryListEvent) {
      setPrefersReducedMotion(!event.matches);
    }
    const mediaQueryList = globalWindow.matchMedia('(prefers-reduced-motion: no-preference)');
    mediaQueryList.addEventListener('change', handleChange);
    
    return () => {
      mediaQueryList.removeEventListener('change', handleChange);
    };
```

Reply:
My guess is he was trying to get around the fact that some browsers don't support this. But given this issue #500 I'd say it doesn't really work as this is the error browsers that don't support this throw:

![error](https://pbs.twimg.com/media/EvBS2GGU4AEOVoK?format=jpg&name=large)

In fact, I don't really understand why the try catch doesn't catch that :think